docs/modules/operation/pages/deep-dive/admin/restart.adoc

```diff
@@ -37,8 +37,8 @@ systemctl stop kibana
 . The {page-component-title} core application (core, Sentinel, Minion), in any order.
 ** Turn off notifications:
 ... Log in to the {page-component-title} web UI.
-... Click the *gear* symbol in the top-right of the page.
-... Under Event Management, turn off *Notification Status* and click *Update*.
+... Click the *gear* symbol at the top-right of the page.
+... Under Event Management, turn toggle Notification Status *Off* and click *Update*.
 ** Access the {page-component-title} system via CLI (putty, SecureCRT, and so on).
```

Comment:
Instead of via CLI, this should probably be ssh or ssh client? Or even favorite ssh client?

Reply:
We use CLI elsewhere throughout the docs, so I think this should be fine. Maybe we could consider adding a glossary entry, though, if it's not an industry standard acronym?

Reply:
Huh, I wasn't aware that it was already generalized as CLI elsewhere in the documentation, it's just that putty and SecureCRT are ssh clients, so to generalize it as a ssh connection via a ssh capable client made more sense to me. But if it is already referenced as CLI everywhere else, then I wouldn't want to change it.

Like you said, it might be worthwhile to add it to the glossary. The CLI is the end destination when using putty, SecureCRT, or even ssh for Linux/macOS/Windows to connect via ssh to the CLI.

Reply:
Created DP-17 for this.